core/templates/dev/head/base.html

```diff
@@ -117,11 +117,11 @@
               <nav class="navbar navbar-default oppia-navbar oppia-prevent-selection" role="navigation">
                 <div class="navbar-container">
                   <div class="navbar-header protractor-test-navbar-header pull-left">
-                    <a ng-click="openSidebar()" class="navbar-brand oppia-navbar-menu oppia-transition-200">
+                    <a ng-if="windowWidth < 768" ng-click="openSidebar()" class="navbar-brand oppia-navbar-menu oppia-transition-200">
                       <img src="/images/general/menu.png">
                     </a>
                     <a class="oppia-navbar-brand-name oppia-transition-200" href="/gallery" focus-on="<[LABEL_FOR_CLEARING_FOCUS]>">
-                      <img src="/images/logo/288x128_logo_white.png" class="oppia-logo">
+                      <img src="/images/logo/288x128_logo_white.png" ng-class="windowWidth > 768 ? 'oppia-logo-wide' : 'oppia-logo-small'">
```

Comment:
Or, maybe better: have two classes here. One is oppia-logo and it's just a regular class. The other is oppia-logo-wide / oppia-logo-small, and it contains just the margin-left specification.